Classes/DDASLLogCapture.m

```diff
@@ -134,11 +132,11 @@ + (void)aslMessageRecieved:(aslmsg)msg {
 
     //  NSString * sender = [NSString stringWithCString:asl_get(msg, ASL_KEY_SENDER) encoding:NSUTF8StringEncoding];
     NSString *message = @(messageCString);
-    NSString *secondsStr = @(secondsCString);
-    NSString *nanoStr = @(nanoCString);
 
-    NSTimeInterval seconds = [secondsStr doubleValue];
-    NSTimeInterval nanoSeconds = [nanoStr doubleValue];
+    const char* secondsCString = asl_get( msg, ASL_KEY_TIME );
+    const char* nanoCString = asl_get( msg, ASL_KEY_TIME_NSEC );
+    NSTimeInterval seconds = secondsCString ? strtod(secondsCString, NULL) : [NSDate timeIntervalSinceReferenceDate] - NSTimeIntervalSince1970;
+    double nanoSeconds = nanoCString? strtod(nanoCString, NULL) : 0;
     NSTimeInterval totalSeconds = seconds + (nanoSeconds / 1e9);
 
```

Comment:
One thing which I did in my custom version of this was to use "asl_log" as the function name instead of nil -- our formatter does not use filenames in the output, but does use function names, so that gives a hint as to where it came from.  I could add that here if that is desirable.